docs/install/command-line.md

```diff
@@ -89,43 +89,72 @@ The IPFS team manages the [dist.ipfs.tech website](https://dist.ipfs.tech/) to h
 
 ### macOS
 
-:::tip M1-based Macs
-You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead of the `amd64` binary listed in these instructions.
-:::
+1. Decide which macOS binary to download from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo) by determining whether your system uses an Apple or Intel CPU. On most macOS systems, you can determine the system specs by doing the following:
 
-1. Download the macOS binary from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo).
+   1. In the upper left hand corner of your screen, click the "Apple" icon. 
+   1. In the drop-down menu displayed, select *About this Mac*.
+   1. A window with information about your Mac is displayed. 
+      - If your system uses Apple Silicon, the specific chip is shown, such as *Apple M1 Pro*.
+      - If your system uses an Intel CPU, the specific processor is shown, such as *2.3 GHz 8-Core Intel Core i9*.
+
+
+1. Download the appropriate macOS binary from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo) based on your hardware. For example:
 
    ```bash
    curl -O https://dist.ipfs.tech/kubo/v0.17.0/kubo_v0.17.0_darwin-amd64.tar.gz
    ```
 
-1. Unzip the file:
+   > :warning:
+   > Ensure that you download and install the appropriate binary, as the binary for an Intel-based system will not work on a system with Apple Silicon, and vice-versa.
+
+   - *If you are using hardware with Apple Silicon, download the `darwin-arm64` binary.* For example, to download the IPFS binary for `Kubo v0.15.0` for an Apple-based system, run the following command:
+     ```bash
+     curl -O https://dist.ipfs.tech/kubo/v0.15.0/kubo_v0.15.0_darwin-arm64.tar.gz
+     ```
+
+   - *If you are using hardware with an Intel Processor, download the `darwin-amd64` binary.* For example, to download the IPFS binary for `Kubo v0.15.0` for an Intel-based system, run the following command:
+     ```bash
+     curl -O https://dist.ipfs.tech/kubo/v0.15.0/kubo_v0.15.0_darwin-amd64.tar.gz
+     ```
+
+1. Unzip the file. For example, to unzip `Kubo v0.15.0` for an Intel-based system:
 
    ```bash
    tar -xvzf kubo_v0.17.0_darwin-amd64.tar.gz
+   ```
```

Comment:
The step says "... to unzip `kubo v0.15.0`..." but then the code block calls out `kubo_v0.17.0...`.

This appears a couple other times in here as well, should probably make them all consistent. I think making them all `kubo v0.17.0` would make most sense, as that's the most recent stable release of Kubo.

Reply:
@TMoMoreau  ty sir good eye, removed mentions of v 15 in the mac install instructions in https://github.com/ipfs/ipfs-docs/pull/1289/commits/88303dadbf1462f543688e76bf2c5270624549d5

Reply:
missed a few https://github.com/ipfs/ipfs-docs/pull/1289/commits/a642879e07a436ddfebb51b03b4d648f8e5cbb17